packages/go_router/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## NEXT
+## 4.3.0
 
-- Cleans up examples
```

Comment:
This seems to be lost

Reply:
Fixed: https://github.com/flutter/packages/pull/2453/commits/87c16ad24a825d9cb68573a20fae1150c5786874